### ISSUE TITLE
Tab look regression in versions 126.* (after 125.0.3)

### ISSUE THREAD

**Issue by Monsterovich** [black7375/Firefox-UI-Fix#943]
### Describe the bug

This is version 128.0.

![изображение](https://github.com/black7375/Firefox-UI-Fix/assets/3750982/e2bf8aaf-757d-4b56-999d-0fb0488e1264)

### Expected behavior

That's how it looked in 125.0.3
Firefox-UI-Fix version: [v8.5.1](https://github.com/black7375/Firefox-UI-Fix/releases/tag/v8.5.1) (I think)

![test](https://github.com/black7375/Firefox-UI-Fix/assets/3750982/9ba50657-83ec-448e-82dc-dae6a7568b29)

### Screenshots

Above.

### OS

Linux

### Firefox Version

128.0

### Distribution

- [ ] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
- [X] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
- [ ] [Lepton's proton style](https://github.com/black7375/Firefox-UI-Fix/tree/proton-style)

### Theme

- [ ] Light
- [ ] Dark
- [ ] Alpenglow (System: Light)
- [ ] Alpenglow (System: Dark)
- [X] System Default
- [ ] Others: (e.g. [Photon Colors](https://addons.mozilla.org/en-US/firefox/addon/photon-colors/), [Google Chrome Dark](https://addons.mozilla.org/en-US/firefox/addon/google-chrome-dark/), [Fairytale Of Nature](https://addons.mozilla.org/en-US/firefox/addon/fairytale-of-nature/))

### Theme - More Info

Marwaita (old version)

### `user.js` setup

Let me just upload the whole chrome dir.

[chrome.zip](https://github.com/user-attachments/files/16161573/chrome.zip)

**Comment by Monsterovich** [comment]
I tried to correct the visual appearance through the options in about:config, but I couldn't find the right ones.

**Comment by Aleksandra0056** [comment]
I'm not using the Photon style, but I honestly think, that you found an important bug.

The outlines should indeed be shown on the borders of tabs and panels. I think, that other people should also tell if they experience this issue. I can test it on a virtual machine to confirm it, but it would take some minutes.

Here could be responses to why this issue happens, but I don't understand these developer notes since I'm not a developer:
https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/126
https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/127

**Comment by Monsterovich** [comment]
@Aleksandra0056 

> I'm not using the Photon style, but I honestly think, that you found an important bug.

Yeah, I first upgraded Firefox on my computer with KDE Plasma 6 and noticed this problem but thought it was an intentional change, then I saw the same thing on my machine with xfce4.

**Comment by Monsterovich** [comment]
I think it has something to do with the `userChrome.tab.static_separator.selected_accent` option, only the color was lighter then.
